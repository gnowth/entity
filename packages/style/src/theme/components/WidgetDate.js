import { css } from 'styled-components';

import { mixin } from '../../selectors';

// eslint-disable-next-line import/prefer-default-export
export const component_widgetDate_main = {
  input: css`
    ${mixin({ name: 'componentBox' })}
  `,
};
