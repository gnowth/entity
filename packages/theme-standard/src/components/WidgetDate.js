import { mixins } from '@gnowth/theme';
import { css } from 'styled-components';

// eslint-disable-next-line import/prefer-default-export
export const component_widgetDate_standard = {
  input: css`
    ${mixins.borderInput}
  `,
};
