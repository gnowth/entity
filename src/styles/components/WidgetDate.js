import { css } from 'styled-components';

export default {
  input: css`
    ${props => props.theme.mixins.componentBox}
  `,
};
