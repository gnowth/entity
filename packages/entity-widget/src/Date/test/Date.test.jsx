import 'jest-styled-components';
import renderer from 'react-test-renderer';
import React from 'react';

import WidgetDate from '../Date';

it('renders correctly', () => {
  const tree = renderer.create(
    <WidgetDate
      name="test"
      onChange={() => undefined}
    />,
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
