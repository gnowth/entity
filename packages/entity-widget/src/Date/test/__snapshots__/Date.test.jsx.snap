// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c1 {
  display: block;
  width: 100%;
}

.c0 .react-datepicker-wrapper {
  display: block;
}

.c0 .react-datepicker__input-container {
  display: block;
}

<div
  className="c0"
>
  <div
    className="react-datepicker-wrapper"
  >
    <div
      className="react-datepicker__input-container"
    >
      <input
        className="c1"
        disabled={false}
        onBlur={[Function]}
        onChange={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        readOnly={false}
        type="text"
        value=""
      />
    </div>
  </div>
</div>
`;
